rbac/management/audit_log/model.py

```diff
@@ -98,6 +97,22 @@ def find_edited_field(self, resource, resource_name, request, object):
                 description = description + "edited access (permissions/resources)"
         return description
 
+    def find_specific_list_of_users(self, type_dict, user_type):
+        """Create list of principals/roles/service accounts for description."""
+        names_list = []
+        if user_type == "user" or user_type == "User":
+            for i in type_dict:
```

Comment:
Nit: can we use a more meaningful name to replace these "i" here?
Such as principal_data, role_data, etc.